cloudify_cli/commands/blueprints.py

```diff
@@ -171,8 +174,12 @@ def upload(ctx,
                 )
                 shutil.rmtree(temp_directory)
 
-    logger.info("Blueprint uploaded. The blueprint's id is {0}".format(
-        blueprint_obj.id))
+    logger.info("Blueprint `{0}` upload started.".format(blueprint_id))
+
+    if not async_upload:
+        blueprint_obj = utils.wait_for_blueprint_upload(client, blueprint_id)
```

Comment:
Do we need to assign this to a variable? Can't we use the `blueprint_id` in the log? 
Or am I missing something?

Reply:
basically another check that we've got a healthy object. It has been this way originally too